nucleus/model_run.py

```diff
@@ -122,12 +127,25 @@ def _format_prediction_response(
         self, response: dict
     ) -> Union[dict, List[Union[BoxPrediction, PolygonPrediction]]]:
         annotations = response.get(ANNOTATIONS_KEY, None)
+        print(annotations)
         if annotations:
-            return [
-                BoxPrediction.from_json(ann)
-                if ann["type"] == "box"
-                else PolygonPrediction.from_json(ann)
-                for ann in annotations
-            ]
+            annotation_response = {}
```

Comment:
Could also clean this up if we make the above suggested change

Reply:
I wasn't able to use the base class approach for the predictions case because there's currently no way to differentiate a segmentation_prediction from a segmentation_gt_annotation from the payloads.  can fix this in the future but don't think it should block this PR.

Reply:
Sgtm